config/config.yaml

```diff
@@ -144,11 +144,3 @@ presets:
     env:
       - name: GOLANG_JUNIT
         value: "true"
-
-  # Istio set this inside their CI builder image and their scripts rely on this
```

Comment:
Do we no longer need to run Istio CI stuff ?

Reply:
I believe Prow now sets this automatically (vaguely remember hearing something along those lines), so we don't need to have an explicit preset for it.

Reply:
Ah, ok! I'm happy with this then :)

Reply:
Found the commit that added it https://github.com/kubernetes/test-infra/commit/c6f0e08ab3. Initially thought that the value of the variable didn't matter, but it actually seems like it does: https://github.com/pusher/istio/blob/pusher-release-1.2/prow/lib.sh#L34